app/models/stripe-account.js

```diff
@@ -0,0 +1,16 @@
+import Model from 'ember-data/model';
+import attr from 'ember-data/attr';
+import { belongsTo } from 'ember-data/relationships';
+
+export default Model.extend({
+  businessName: attr(),
```

Comment:
Is there a reason we don't specify these as `string` transforms but we do specify for `boolean` and `date`?